.github/workflows/flutter.yml

```diff
@@ -89,8 +89,6 @@ jobs:
             min_coverage: 90
 
         - name: Build ${{ matrix.target }}
-          env:
-            SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
```

Comment:
Shouldn't this be kept? Otherwise, the E2E tests doesn't work any more even if `SENTRY_AUTH_TOKEN` is available.

Reply:
Should I put this again?

Reply:
@ueman why do we need this? if `SENTRY_AUTH_TOKEN` is given in the e2e_dart.yml? I don't see it being used.

Reply:
Oh sorry, never mind then.